packages/Telephony/Actions/TimeoutInput.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Input;
+using Microsoft.Bot.Components.Telephony.Common;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Components.Telephony.Actions
+{
+    public static class TimeoutInput
+    {
+        // Summary:
+        //     Defines dialog context state property value.
+        private const string TimerID = "dialog.TimerId";
+        //private const string TimeoutId = "this.TimeoutId";
+        //private const string ActiveTimeoutId = "conversation.ActiveTimeoutId";
+
+        private static ConcurrentDictionary<string, (ConcurrentQueue<string> triggeredTimers, IStateMatrix timersState)> conversationStateMatrix = new ConcurrentDictionary<string, (ConcurrentQueue<string> triggeredTimers, IStateMatrix timersState)>();
```

Comment:
Will this work in a setup with multiple instances?

Reply:
By multiple instances means multiple nodes one per each reigon? 
I believe each instance is its own app and runs individually and shouldn't affect each other, am I right?